source/core/Date.cpp

```diff
@@ -250,17 +251,18 @@ namespace Vireo {
         struct tm tm;
         time_t now = time(NULL);
         localtime_r(&now, &tm);
-        _SystemLocaletimeZone = int(tm.tm_gmtoff);
+        _systemLocaleTimeZone = int(tm.tm_gmtoff);
 #else
         // flipping the sign of the time zone
         TIME_ZONE_INFORMATION timeZoneInfo;
         GetTimeZoneInformation(&timeZoneInfo);
-        _SystemLocaletimeZone = -int((timeZoneInfo.Bias + timeZoneInfo.DaylightBias)* 60);
+        _systemLocaleTimeZone = -int((timeZoneInfo.Bias + timeZoneInfo.DaylightBias) * secondsPerMinute);
 #endif
-        return _SystemLocaletimeZone;
+        return _systemLocaleTimeZone;
     };
+
     //------------------------------------------------------------
-    Int32 Date::isDTS()
+    Int32 Date::isDayTimeSaving()
```

Comment:
It is commonly referred to as DaylightSavingTime (or abbreviated as DST) and I would go with either that full name or abbreviation.